pandas/tests/io/pytables/common.py

```diff
@@ -45,7 +32,7 @@ def ensure_clean_store(
 
         # put in the temporary path if we don't have one already
         if not len(os.path.dirname(path)):
-            path = create_tempfile(path)
+            path = os.path.join(tempfile.gettempdir(), path)
```

Comment:
Just asking: Is there a reason for using os instead of pathlib? Would prefer the latter because of platform independency

Reply:
Probably not; this was just copying over code from `create_tempfile`. Willl use pathlib here